xml/Microsoft.Extensions.Configuration/IConfiguration.xml

```diff
@@ -114,7 +114,9 @@
         <summary>Gets a configuration sub-section with the specified key.</summary>
         <returns>The <see cref="T:Microsoft.Extensions.Configuration.IConfigurationSection" />.</returns>
         <remarks>This method will never return <see langword="null" />. If no matching sub-section is found with the specified key,
-                an empty <see cref="T:Microsoft.Extensions.Configuration.IConfigurationSection" /> will be returned.</remarks>
+                an empty <see cref="T:Microsoft.Extensions.Configuration.IConfigurationSection" /> will be returned.
+        
+                You can reach sub-subsection by passing "section1:section2:section.." as key parameter</remarks>
```

Comment:
I think the grammar may be off a bit here.

```suggestion
                You can reach a sub-subsection by passing "section1:section2:section.." for <paramref name="key"/>.</remarks>
```

?

Reply:
Yes, this is right, thank you